Sources/Rendering/OpenGL/RenderWindow/index.js

```diff
@@ -1,16 +1,20 @@
+import { VtkDataTypes } from 'vtk.js/Sources/Common/Core/DataArray/Constants';
 import macro from 'vtk.js/Sources/macros';
 import { registerViewConstructor } from 'vtk.js/Sources/Rendering/Core/RenderWindow';
 import vtkForwardPass from 'vtk.js/Sources/Rendering/OpenGL/ForwardPass';
+import vtkOpenGLHardwareSelector from 'vtk.js/Sources/Rendering/OpenGL/HardwareSelector';
+import vtkShaderCache from 'vtk.js/Sources/Rendering/OpenGL/ShaderCache';
+import vtkOpenGLTextureUnitManager from 'vtk.js/Sources/Rendering/OpenGL/TextureUnitManager';
 import vtkOpenGLViewNodeFactory from 'vtk.js/Sources/Rendering/OpenGL/ViewNodeFactory';
 import vtkRenderPass from 'vtk.js/Sources/Rendering/SceneGraph/RenderPass';
-import vtkShaderCache from 'vtk.js/Sources/Rendering/OpenGL/ShaderCache';
 import vtkRenderWindowViewNode from 'vtk.js/Sources/Rendering/SceneGraph/RenderWindowViewNode';
-import vtkOpenGLTextureUnitManager from 'vtk.js/Sources/Rendering/OpenGL/TextureUnitManager';
-import vtkOpenGLHardwareSelector from 'vtk.js/Sources/Rendering/OpenGL/HardwareSelector';
-import { VtkDataTypes } from 'vtk.js/Sources/Common/Core/DataArray/Constants';
+
+// import WebXRPolyfill from 'webxr-polyfill';
+
+// const polyfill = new WebXRPolyfill();
```

Comment:
Leftovers?

Reply:
Yes, will remove here. @floryst and I discussed the WebXR polyfill briefly today, will subsequently explore adding the polyfill as an external dependency.